### ISSUE TITLE
Version 2.3.1 raising error when using connectionString: $([Variable Name])

### ISSUE THREAD

**Issue by mauriciomagni** [cmeeren/Facil#38]
After updating to version 2.3.1, it started raising an error when trying to generate DbGen file, saying:
facil.yaml(0 , 0) : error : The variable $(Db) could not be found in the specified configuration sources

Going back to version 2.3.0 works as expected.

In my facil.yaml file I have this configuration:
```
rulesets:
  - connectionString: $(Db)
```

**Comment by cmeeren** [comment]
I can't reproduce. The following works fine in Facil's test project:

```yaml
configs:
  - appSettings: appsettings.json

rulesets:
  - connectionString: $(connectionString)
```

I need a minimal repro. Could you create one?

**Comment by cmeeren** [comment]
Also, please verify that the variable casing is correct.

**Comment by mauriciomagni** [comment]
hello Cmeeren
The casing is correct, and we have been using it until this new version.
I will create a small project to try to reproduce this error.
Something I did not mentioned before is the "configs:" configuration:
`configs:
  - envVars: MyApp__
`
I will get back to you with minimal repro.
Kind regards,
Mauricio

**Comment by mauriciomagni** [comment]
[Facil231.zip](https://github.com/cmeeren/Facil/files/8176748/Facil231.zip)
Dear Cmeeren
Attached is the minimal repro.
Here is how I have configure the environment variable:
![image](https://user-images.githubusercontent.com/20417121/156540452-1852015d-41c6-4dc4-b2ac-3815afcd164f.png)
Hope this help.
If you cannot look at it, I will try to create a pull request during the weekend.
Regards,
Mauricio

**Comment by cmeeren** [comment]
Thanks, I'll have a look at it. If you happen to know where the bug is, don't hesitate to let me know.

**Comment by cmeeren** [comment]
Related to https://github.com/dotnet/runtime/issues/65756

I'll downgrade if there's no new fix published.

**Comment by cmeeren** [comment]
Fixed in v/2.3.2.
